.github/workflows/ci.yml

```diff
@@ -32,30 +32,68 @@ jobs:
         run: yarn build:dev
       - name: Run Unit tests
         run: yarn tools --test --unit
-  e2e-connection:
+  integration-tests:
     runs-on: ubuntu-latest
     timeout-minutes: 30
     env:
-      NETWORK: datil-dev
-      DEBUG: true
-      MAX_ATTEMPTS: 3
-      CHUNK_SIZE: 9
+      DATIL_COMMIT_HASH: b5cfe7489275dbed5e9098818d0bd441c8918903
```

Comment:
Since we have now merged the lit-assets PR should we point this to develop?

Reply:
We will want to point it at `datil` for now as that is our current compatibility with `v7` as of now. But once we merge https://github.com/LIT-Protocol/lit-assets/pull/1341 we can switch from the branch used to [datil](https://github.com/LIT-Protocol/lit-assets/tree/datil).


I think once we start on changes for v7 like [this](https://github.com/LIT-Protocol/lit-assets/pull/1372) for example we can use `develop` in a parallel job.